samples/subsys/fs/littlefs/README.rst

```diff
@@ -88,7 +88,7 @@ To build the test:
    :gen-args: -DCONF_FILE=prj_blk.conf
    :compact:
 
-One can also set ``CONFIG_SDMMC_VOLUME_NAME`` to provide the mount point name
+One can also set ``CONFIG_SDMMC_VOLUME_NAME`` or ``CONFIG_MMC_VOLUME_NAME`` (in case of MMC) to provide the mount point name
```

Comment:
I think we should reword this to make it clear that, depending on selected device, the name of device will be used form mount point and will be take from one of above configs. Somehow I do not like the "One can also set (...)" beginning of sentence and would rather have "Mount point is controlled by"

Reply:
got it, I have rewritten this part of the littlefs sample documentation
thank you for a review